utils/src/grpc.rs

```diff
@@ -79,17 +87,17 @@ where
         tokio::spawn(async move {
             while let Some(Ok(chunk)) = body.data().await {
                 debug!("[SIZE MW] request body chunk size = {}", chunk.len());
-                let previous = bytes_sent_counter.fetch_add(chunk.len(), Ordering::Relaxed);
-                debug!("[SIZE MW] total count: {}", previous);
-                if let Err(err) = tx.send_data(chunk).await {
-                    warn!("[SIZE MW] error sending data: {}", err)
+                let _previous = bytes_sent_counter.fetch_add(chunk.len(), Ordering::Relaxed);
+                debug!("[SIZE MW] total count: {}", _previous);
+                if let Err(_err) = tx.send_data(chunk).await {
+                    debug!("[SIZE MW] error sending data: {}", _err)
```

Comment:
This error should never come, although I prefer to have error level here. It will point that something is wrong